src/arrayMethodSort.js

```diff
@@ -5,7 +5,34 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        const strCurrentEl = this[i].toString();
+        const strNextEl = this[j].toString();
+
+        if (compareFunction) {
```

Comment:
create separate function and use it as default value of compareFunciton `function(compareFunction = compareAsStrings) {`